azure-pipelines.yml

```diff
@@ -176,98 +173,162 @@ steps:
       GdnBreakGdnToolCredScanSeverity: Warning
       GdnBreakGdnToolESLint: true
       GdnBreakGdnToolESLintSeverity: Warning
-      GdnBreakPolicy: M365
+      GdnBreakPolicy: M365  
 
-  - task: Yarn@3
+  - task: AzureRmWebAppDeployment@4
     condition: and(
-      startsWith(variables['Build.SourceBranch'], 'refs/tags/v'),
       in(variables['Agent.JobStatus'], 'Succeeded', 'SucceededWithIssues'),
-      ne(variables['Build.Reason'], 'PullRequest'),
-      ne(variables['Build.Reason'], 'Manual')
-      )
+      startsWith(variables['Build.SourceBranch'], 'refs/tags/v'))
     inputs:
-      Arguments: 'publish-sdk'
-    displayName: 'yarn publish-sdk (main)'
-
-  - script: 'node prepNextRelease.js'
-    workingDirectory: '$(System.DefaultWorkingDirectory)/tools/cli'
-    displayName: 'yarn node prepNextRelease.js'
-    name: prepNextRelease
-    
-  - task: CopyFiles@2
-    inputs:
-      sourceFolder: 'packages\teams-js\dist'
-      contents: '**'
-      targetFolder: '$(Build.ArtifactStagingDirectory)\$(prepNextRelease.NextVersion)\js'
-    displayName: 'Copy TeamsJS content'
+      ConnectionType: 'AzureRM'
+      azureSubscription: '$(AzureSubscription)'
+      appType: 'webApp'
+      WebAppName: 'musicalsink'
+      package: '$(Build.ArtifactStagingDirectory)\teams-test-app\$(Build.BuildId).zip'
+    displayName: 'Deploy to musicalsink'
 
-  - task: PublishBuildArtifacts@1
+  - task: AzureRmWebAppDeployment@4
+    condition: and(
+      in(variables['Agent.JobStatus'], 'Succeeded', 'SucceededWithIssues'),
+      eq(variables['Build.SourceBranch'], 'refs/heads/2.0-preview'))
     inputs:
-      PathtoPublish: '$(Build.ArtifactStagingDirectory)\$(prepNextRelease.NextVersion)'
-      ArtifactName: '$(prepNextRelease.NextVersion)'
-    displayName: 'Publish TeamsJS to build Artifacts'
+      ConnectionType: 'AzureRM'
+      azureSubscription: '$(AzureSubscription)'
+      appType: 'webApp'
+      WebAppName: 'cloudroll'
+      package: '$(Build.ArtifactStagingDirectory)\teams-test-app\$(Build.BuildId).zip'
+    displayName: 'Deploy to cloudroll'
 
-  - task: Yarn@3
+  # TODO - delete after release pipeline creation
+  - task: CmdLine@1
     condition: and(
-      eq(variables['Build.SourceBranch'], 'refs/heads/develop'),
+      startsWith(variables['Build.SourceBranch'], 'refs/tags/v'),
       in(variables['Agent.JobStatus'], 'Succeeded', 'SucceededWithIssues'),
       ne(variables['Build.Reason'], 'PullRequest'),
-      ne(variables['Build.Reason'], 'Manual')
-      )
+      ne(variables['Build.Reason'], 'Manual'))
     inputs:
-      Arguments: 'publish-sdk --tag next'
-    displayName: 'yarn publish-sdk (develop)'
+      filename: node
+      arguments: publish.js
+      workingFolder: '$(System.DefaultWorkingDirectory)'
```

Comment:
The workingFolder should be '$(System.DefaultWorkingDirectory)\packages\teams-js'